src/components/Checkout/LoanReservation.vue

```diff
@@ -89,38 +99,42 @@ export default {
 		},
 		reservationMessage() {
 			if (this.expiryTime !== null) {
-				const reservedDate = new Date(this.expiryTime);
-				const mins = differenceInMinutes(reservedDate.getTime(), Date.now());
-				const seconds = differenceInSeconds(reservedDate.getTime(), Date.now()) % 60;
+				const mins = differenceInMinutes(this.reservedDate.getTime(), Date.now());
+				const seconds = differenceInSeconds(this.reservedDate.getTime(), Date.now()) % 60;
 
-				let warningMessageUpperBoundMinutes = 6;
+				let warningMessageUpperBoundMinutes = 5;
 				let differenceInWords = `Reserved for ${mins} more minutes`;
 
 				if (this.activateTimer === true) {
-					warningMessageUpperBoundMinutes = 10;
+					warningMessageUpperBoundMinutes = 9;
```

Comment:
Previously these 2 bound settings were 1 minute off due to the inclusion of seconds. In other words 6:59 and 10:59 would flip the class to red whereas we want it to flip at 5:59 + 9:59.

Reply:
Nice